src/__test/DisplayLists.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import DisplayLists from '../components/DisplayLists';
+
+it('renders correctly', () => {
+  const tree = Renderer.create(
+    <React.StrictMode>
+      <Provider store={store}>
+        <BrowserRouter>
+          <DisplayLists />
+        </BrowserRouter>
+      </Provider>
+    </React.StrictMode>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Please ensure you have more test coverage in this Project, you also need to add Unit tests for the Reducer functions (You can just stub the new State and it is very easy since reducers are pure functions by default). Kindly also add some integration tests for your project to ensure user interactivity between components is as expected. Tests will ensure that your project is robust and less prone to edge case bugs

Reply:
Thank you @zhadier for your review, I will implement all the changes required.